### ISSUE TITLE
The RubyEncoder loader is not installed on vagrant 1.8.4 on windows hosts

### ISSUE THREAD

**Issue by mwrock** [mitchellh/vagrant#7490]
### Vagrant version
1.8.4
### Host operating system
Windows 10 (tried both on RTM and preview build 14366)

### Guest operating system
Windows 2012 R2

### Only when launching via Test-Kitchen
I seem to be able to perform `vagrant up` just fine. However if I run Test-Kitchen using the vagrant provider, I receive this error very early in the run:
```
-----> Creating <default-windows-2012R2>...
       Vagrant failed to initialize at a very early stage:

       The plugins failed to load properly. The error message given is
       shown below.

       The RubyEncoder loader is not installed. Please visit the http://www.rubyencoder.com/loaders/ RubyEncoder site to download the required loader for 'mingw' and unpack it into 'C:/HashiCorp/Vagrant/embedded/rgloader' directory to run this protected script.
```
If I uninstall 1.8.4 and reinstall 1.8.1. This problem goes away. Further if I download the mingw ruby encoder from https://www.rubyencoder.com/loaders.html and extract it into `c:\hashicorp\vagrant\embedded` this seems to fix the 1.8.4 install.

**Comment by sethvargo** [comment]
Hi @mwrock 

Do you have vagrant installed as a gem that TK is picking up? That would explain why you're getting that eror.

**Comment by mwrock** [comment]
no its a normal msi install @sethvargo

**Comment by akemner** [comment]
i was able to reproduce this bug, and the workaround @mwrock describes worked for me too, so thanks for that Matt.

**Comment by chrisroberts** [comment]
@mwrock @akemner Hi! I'm attempting to reproduce this and am so far unable to do so. From the description of the fix, it sounds like vagrant is running under the Ruby shipped with chefdk, but I would imagine there would be other weirdness going on as well. Do you experience the same problem with 1.8.5?

If you're able to reproduce the issue, running it with debug enabled and providing the debug output would be really helpful to track down the issue. Thanks!

**Comment by mwrock** [comment]
Hi @chrisroberts I'll try 1.8.5 but to be clear, I'm not running vagrant as a gem inside of another ruby. I always run it in the omnibus vagrant ruby environment it ships with.

**Comment by mwrock** [comment]
Looks like this is resolved in 1.8.5 thanks!

**Comment by chrisroberts** [comment]
:+1: thanks for following up @mwrock !
